.circleci/bin/perf

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+[[ "$TRACE" ]] && set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o noclobber
+
+URL=${1:-https://search.dev.trade-tariff.service.gov.uk/fpo-code-search}
+API_KEY=${2:-$API_KEY}
+DESIRED_ERRORS=0
+DESIRED_LATENCY=2000
+
+if [ "$API_KEY" = "" ]; then
+  echo "API_KEY is not set"
+  exit 1
+fi
+
+if ! command -v jq >/dev/null 2>&1; then
+  echo "jq is not installed"
+  exit 1
+fi
+
+echo "Running performance test on $URL"
+PERFORMANCE_DATA=$(
+  yarn run autocannon \
```

Comment:
I wonder if we're better off running this straight through `npx` rather than introducing a whole yarn.lock and whatnot just for a single executable?